src/tests/multihost/alltests/test_misc.py

```diff
@@ -478,3 +478,40 @@ def test_0008_1636002(multihost, backupsssdconf):
         assert "Failed to connect to '/var/lib/sss/db/config.ldb'" \
             not in log_str
         assert "The confdb initialization failed" not in log_str
+
+    @pytest.mark.tier1_3
+    def test_0009_gssapi_ssh(self, multihost, backupsssdconf):
+        """
+        :title: gssapi ssh log in with 'krb5_confd_path'
+        :description: User should log in with GSSAPI after setting
+         'krb5_confd_path' option in sssd.conf
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1961182
+        :id: 752b69ab-55a8-464f-814b-4985c06dc49a
+        :customerscenario: true
+        :steps:
+            1. Set 'krb5_confd_path' option in sssd.conf
+            2. restart SSSD with empty cache.
+            3. Fetch kerberos ticket for user
+            4. Check user is able to log in with GSSAPI
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+            4. User should be to log in with GSSAPI
+        """
+        client = sssdTools(multihost.client[0])
+        domain_name = client.get_domain_section_name()
+        section = f"domain/{domain_name}"
+        section_params = {"krb5_confd_path": "/etc/krb5.conf"}
```

Comment:
@ikerexxe, shouldn't this be a directory? So that we dont have to update the krb5.conf with `includedir /var/lib/sss/pubconf/krb5.include.d`
eg.
`krb5_confd_path = /etc/krb5.include.d`

Reply:
The `includedir /var/lib/sss/pubconf/krb5.include.d/` configuration is already installed in a snippet in `/etc/krb5.conf.d/`. So we only need to make sure that `krb5_confd_path` points to this directory. This can be done  in two ways: by setting `krb5_confd_path` to `/etc/krb5.conf.d/` or by setting it to `/etc/krb5.conf` and including `/etc/krb5.conf.d/` from this file.

There's no need to change `krb5.conf` because the configuration snippet already takes charge of it.